Source/ACE.Server/Network/Handlers/AuthenticationHandler.cs

```diff
@@ -71,131 +72,126 @@ private static void DoLogin(Session session, PacketInboundLoginRequest loginRequ
             {
                 if (!account.PasswordMatches(loginRequest.Password))
                 {
-                    if (WorldManager.WorldStatus == WorldManager.WorldStatusState.Open)
-                        log.Info($"client {loginRequest.Account} connected with non matching password does so booting");
-                    else
-                        log.Debug($"client {loginRequest.Account} connected with non matching password does so booting");
-
+                    log.Info($"client {loginRequest.Account} connected with non matching password so booting");
                     session.Terminate(SessionTerminationReason.NotAuthorizedPasswordMismatch, new GameMessageCharacterError(CharacterError.AccountDoesntExist));
-
-                    // TO-DO: temporary lockout of account preventing brute force password discovery
-                    // exponential duration of lockout for targeted account
-
                     return;
                 }
-
-                if (WorldManager.WorldStatus == WorldManager.WorldStatusState.Open)
-                    log.Info($"client {loginRequest.Account} connected with verified password");
-                else
-                    log.Debug($"client {loginRequest.Account} connected with verified password");
+                session.SetAccount(account.AccountId, account.AccountName, (AccessLevel)account.AccessLevel);
+                session.State = SessionState.AuthConnectResponse;
+                log.Info($"session {session} connected with verified password");
             }
             else if (loginRequest.NetAuthType == NetAuthType.GlsTicket)
             {
                 if (WorldManager.WorldStatus == WorldManager.WorldStatusState.Open)
                     log.Info($"client {loginRequest.Account} connected with GlsTicket which is not implemented yet so booting");
                 else
                     log.Debug($"client {loginRequest.Account} connected with GlsTicket which is not implemented yet so booting");
-
                 session.Terminate(SessionTerminationReason.NotAuthorizedGlsTicketNotImplementedToProcLoginReq, new GameMessageCharacterError(CharacterError.AccountInvalid));
-
                 return;
             }
-
-            // TODO: check for account bans
-
-            session.SetAccount(account.AccountId, account.AccountName, (AccessLevel)account.AccessLevel);
-            session.State = SessionState.AuthConnectResponse;
         }
-
         public static void HandleConnectResponse(Session session)
         {
             if (WorldManager.WorldStatus == WorldManager.WorldStatusState.Open || session.AccessLevel > AccessLevel.Player)
             {
                 DatabaseManager.Shard.GetCharacters(session.AccountId, false, result =>
                 {
                 // If you want to create default characters for accounts that have none, here is where you would do it.
+                if (result.Count == 0)
```

Comment:
Think everything in this func isn't meant for master...

Reply:
woops, left that in by accident, thanks for the note, i'll remove it.